native/src/components/__tests__/Header.spec.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { fireEvent, render } from '@testing-library/react-native'
+
+import CityModelBuilder from 'api-client/src/testing/CityModelBuilder'
+import LanguageModelBuilder from 'api-client/src/testing/LanguageModelBuilder'
+import buildConfig from '../../constants/buildConfig'
+import Header from '../Header'
+import createNavigationScreenPropMock from '../../testing/createNavigationPropMock'
+import { SEARCH_ROUTE } from 'api-client'
+
+describe('Header', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  const t = key => key
+  const goToLanguageChange = jest.fn()
+  const dispatch = jest.fn()
+  const [city] = new CityModelBuilder(1).build()
+  const language = new LanguageModelBuilder(1).build()[0]
+  const navigation = createNavigationScreenPropMock()
+
+  const buildProps = (peeking: boolean, categoriesAvailable: boolean, mode: 'float' | 'screen', goToLanguageChange) => {
```

Comment:
you can also just use one ts-ignore if you specify the return type of `buildProps`
```suggestion
  const buildProps = (peeking: boolean, categoriesAvailable: boolean, mode: 'float' | 'screen', goToLanguageChange): React.ComponentProps<typeof Header> => {
    // @ts-ignore StackHeaderProps not needed
```